### PR TITLE
49302_rspec_courses_subject_model

### PR DESCRIPTION
**Pull request by zenith19**

### CODE REVIEW

**Comment by Rownaknira** [comment]
ok

File: app/models/courses_subject.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@ class CoursesSubject < ActiveRecord::Base
 
   attr_accessor :update_status
 
+  validates :course_id, presence: true
```
Comment (dieunb) [inline app/models/courses_subject.rb]: How do you think about validate with `course` or `course_id`

Comment (zenith19) [inline app/models/courses_subject.rb]: sorry, it was a mistake

File: spec/factories/courses_subject.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require "faker"
+
+FactoryGirl.define do
+  factory :courses_subject do |f|
+    f.course_id {Faker::Number.number}
```
Comment (dieunb) [inline spec/factories/courses_subject.rb]: Should get it from Course object

File: spec/factories/courses_subject.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    f.subject_id {Faker::Number.number}
```
Comment (dieunb) [inline spec/factories/courses_subject.rb]: Subject object

**Pull request merged**